### PR TITLE
Added tests and perf improvements for Bigtable CDC->BigQuery

### PR DESCRIPTION
**Pull request by alexeykukuku**
Warning: 

This PR contains dependency versions overrides, we need to talk about it

### CODE REVIEW

File: v2/googlecloud-to-googlecloud/pom.xml [hunk 1]
```diff
@@ -62,6 +62,30 @@
...
+        <artifactId>google-cloud-bigtable</artifactId>
+        <version>2.25.0</version>
+        <type>jar</type>
+      </dependency>
+      <dependency>
+        <groupId>com.google.api</groupId>
+        <artifactId>gax</artifactId>
+        <version>2.31.1</version>
+        <type>jar</type>
+      </dependency>
```
Comment (bvolpato) [inline v2/googlecloud-to-googlecloud/pom.xml:79]: Can you use 2.31.0 please? Based on https://mvnrepository.com/artifact/com.google.cloud/google-cloud-bigtable/2.25.0

And add a TODO to remove this block once libraries-bom gets past `26.19.0`? (likely Beam 2.50.0)

File: it/common/src/main/java/com/google/cloud/teleport/it/common/utils/ExceptionMessageUtils.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    while (t != null) {
+      String msg = t.getMessage();
+      if (msg != null && msg.contains(messageToLookFor)) {
+        return true;
+      } else {
+        t = t.getCause();
+      }
+    }
+    return false;
+  }
```
Comment (bvolpato) [inline it/common/src/main/java/com/google/cloud/teleport/it/common/utils/ExceptionMessageUtils.java:35]: Duplicated, please see `ExceptionUtils.containsMessage`

**Comment by alexeykukuku** [comment]
I will create another pull request

**Comment by alexeykukuku** [comment]
This pull request is superseded by https://github.com/GoogleCloudPlatform/DataflowTemplates/pull/925

**Pull request closed without merging**